.github/ISSUE_TEMPLATE/mass-name-squatter.yml

```diff
@@ -0,0 +1,46 @@
+---
+name: "Mass name squat by user: USERNAME"
+title: "Mass name squat by user: USERNAME"
+description: You want to report mass project name squatting by a PyPI user
+labels: mass name squat
+
+body:
+- type: markdown
+  attributes:
+    value: |
+      A PyPI user has registered a large number of projects with what you
+      believe is an attempt to reserve their names or otherwise abuse the
+      service.
+      
+- type: input
+  attributes:
+    label: PyPI user performing the mass project name squatting
+    description: |
+      Please also **update the issue title** as well.
+    value: |
+      https://pypi.org/user/USERNAME
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: Reasons for the request
+    description: |
+      Why do you think the user is engaging in mass project name squatting?
```

Comment:
The person filling this form would not have enough context to answer this IMO.

Reply:
Yeah, it's probably phrased incorrectly, my intention was less "what's the squatter's end goal?" and more "have you noticed a pattern? are most projects empty?", I think I'll change it to a more open "additional information".